core/server/proxy/src/main/java/alluxio/proxy/s3/S3ErrorCode.java

```diff
@@ -49,6 +50,10 @@ private Name() {
   //
   // Official error codes.
   //
+  public static final S3ErrorCode ACCESS_DENIED_ERROR = new S3ErrorCode(
+      Name.ACCESS_DENIED_ERROR,
+      "User doesn't have the right to access this resource",
+      Response.Status.FORBIDDEN);
```

Comment:
Manually ported over from https://github.com/Alluxio/alluxio/pull/15164
- We do not wish to carry-over this PR as it affects how the "Authorization" HTTP header is parsed in the S3 API